src/lib/markdown.ts

```diff
@@ -9,20 +9,22 @@ const LINE_SEPARATOR = /\u2028/g
 // It replaces line breaks with a visual symbol (↵) to indicate where new lines exist in the original text.
 const LINE_BREAK_VISUAL = '↵ '
 
-marked.setOptions({
+const markedInstance = new Marked()
```

Comment:
Suggestion: This is also fine. Resolve if you leave as is.

```suggestion
const marked = new Marked()
```